protocol-designer/src/components/modals/CreateFileWizard/ModulesAndOtherTile.tsx

```diff
@@ -169,14 +169,18 @@ export function ModulesAndOtherTile(props: WizardTileProps): JSX.Element {
         >
           <GoBack
             onClick={() => {
-              if (
-                props.values.pipettesByMount.left.pipetteName === 'p1000_96'
-              ) {
-                goBack(3)
-              } else if (
-                props.values.pipettesByMount.right.pipetteName === ''
-              ) {
-                goBack(2)
+              if (!enableDeckModification || robotType === OT2_ROBOT_TYPE) {
+                if (
+                  props.values.pipettesByMount.left.pipetteName === 'p1000_96'
```

Comment:
I think you can remove props since you destruct props in the above.
```suggestion
                  values.pipettesByMount.left.pipetteName === 'p1000_96'
```